app/models/user.rb

```diff
@@ -307,6 +307,24 @@ def self.find_by_username(username)
     find_by(username_lower: username.downcase)
   end
 
+  # TODO: To be removed after column renaming has succesfully completed.
```

Comment:
Sorry this is so hairy, there is a lot to learn here

I am a bit confused about this. when this code runs always ` has_attribute?(:group_min_trust_level)` will be around. All we really want here is something that makes sure we ignore the old column.